2.0/ingest/ingest_logic.py

```
@@ -0,0 +1,67 @@
+    '''
+    Testing, new method with handling of PK conflicts.
+    '''
+    df = serialize_nonbasic_df_columns(df)
+    model = TABLE_CLASS_MAP.get(table_name)
+    with engine.begin() as conn:
+        for row in df.to_dict(orient='records'):
+            stmt = insert(model).values(**row)
+            stmt = stmt.on_conflict_do_nothing(index_elements=['accession_number']) 
+            try:
+                conn.execute(stmt)
+            except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)